Mod.cs

```diff
@@ -78,77 +81,77 @@ public class RandomColorWidgetState
+        //    api.RegisterPageLabel( ModManifest, "Back to main page", "", "" );
+        //    api.RegisterSimpleOption(ModManifest, "Dummy String (1)", "Testing a string", () => config.dummyString1, (string val) => config.dummyString1 = val);
+        //    api.RegisterChoiceOption(ModManifest, "Dummy String (2)", "Testing a dropdown box", () => config.dummyString2, (string val) => config.dummyString2 = val, DummyConfig.dummyString2Choices);
+        //    api.RegisterSimpleOption(ModManifest, "Dummy Keybinding", "Testing a keybinding", () => config.dummyKeybinding, (SButton val) => config.dummyKeybinding = val);
+        //    api.RegisterSimpleOption(ModManifest, "Dummy Keybinding 2", "Testing a keybinding list", () => config.dummyKeybinding2, (KeybindList val) => config.dummyKeybinding2 = val);
+
+        //    api.RegisterLabel(ModManifest, "", "");
+
+        //    // Complex widget - this just generates a random  color on click.
+        //    Func<Vector2, object, object> randomColorUpdate =
+        //    (Vector2 pos, object state_) =>
+        //    {
+        //        var state = state_ as RandomColorWidgetState;
+        //        if (state == null)
+        //            state = new RandomColorWidgetState() { color = config.dummyColor };
+
+        //        var bounds = new Rectangle((int)pos.X + 12, (int)pos.Y + 12, 50 - 12 * 2, 50 - 12 * 2);
+        //        bool hover = bounds.Contains(Game1.getOldMouseX(), Game1.getOldMouseY());
+        //        if ( hover && Game1.oldMouseState.LeftButton == ButtonState.Released && Mouse.GetState().LeftButton == ButtonState.Pressed)
+        //        {
+        //            Game1.playSound("drumkit6");
+        //            Random r = new Random();
+        //            state.color.R = (byte) r.Next(256);
+        //            state.color.G = (byte) r.Next(256);
+        //            state.color.B = (byte) r.Next(256);
+        //        }
+
+        //        return state;
+        //    };
+        //    Func<SpriteBatch, Vector2, object, object> randomColorDraw =
+        //    (SpriteBatch b, Vector2 pos, object state_) =>
+        //    {
+        //        var state = state_ as RandomColorWidgetState;
+        //        IClickableMenu.drawTextureBox(b, (int)pos.X, (int)pos.Y, 50, 50, Color.White);
+        //        var colorBox = new Rectangle((int)pos.X + 12, (int)pos.Y + 12, 50 - 12 * 2, 50 - 12 * 2);
+        //        b.Draw(Game1.staminaRect, colorBox, state.color);
+        //        return state;
+        //    };
+        //    Action<object> randomColorSave =
+        //    (object state) =>
+        //    {
+        //        if ( state == null )
+        //            return;
+        //        config.dummyColor = (state as RandomColorWidgetState).color;
+        //    };
+        //    api.RegisterComplexOption(ModManifest, "Dummy Color", "Testing a complex widget (random color on click)", randomColorUpdate, randomColorDraw, randomColorSave);
+        //}
```

Comment:
Commented out this whole func (and remove ref to it in `Entry`) because it's just a sample.